skills/image-explore/generate.py

```
@@ -254,6 +262,43 @@ def _scan_border_for_magenta_seeds(magick_bin, image_path, w, h):
+    Recraft regresses, the alpha-mean / interior-hole signals catch it.
+
+    See bead igor2-88g.61 for smoke-test results (alpha-mean within 0.2%
+    of magenta, cleaner edges, smaller files).
+    """
+    if not recraft_script or not Path(recraft_script).exists():
+        return False, (
+            f"recraft script not found at {recraft_script!r}; "
+            "expected skills/gen-image/recraft-bg-remove.sh"
+        )
+
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)